Libraries/LibWeb/CSS/Default.css

```diff
@@ -857,3 +857,40 @@ progress {
         filter: invert(100%);
     }
 }
+
+/* https://github.com/whatwg/html/pull/9546
+ */
+input[type=checkbox][switch] {
+	appearance: none;
+	height: 1em;
+	width: 1.8em;
+	border: 1px solid;
```

Comment:
You don't need to set a border to use `border-radius`. This seems unnecessary? (If it is necessary, merge the `border-color: transparent` below into it.

Reply:
> You don't need to set a border to use `border-radius`. This seems unnecessary? (If it is necessary, merge the `border-color: transparent` below into it.

Not necessary — removed now